src/components/Configuration/Chart/ChartToogle.tsx

```diff
@@ -0,0 +1,25 @@
+import { Switch } from "@headlessui/react";
+import { useTaoVoting } from "../../../hooks/useTaoVoting";
+
+export function ChartToogle(){
+    const { isChartOpen, setIsChartOpen } = useTaoVoting()
+    return (
+        <div className="flex w-full justify-center">
+            <p>Open chart view</p>
+            <Switch
+            checked={isChartOpen}
+            onChange={setIsChartOpen}
+            className={`${
+            isChartOpen ? 'bg-blue-600' : 'bg-gray-200'
+            } relative inline-flex h-6 w-11 items-center rounded-full`}
```

Comment:
Hey Lu, please take a look on the classnames function to handle conditional CSS!